### PR TITLE
Add .projs for new SDK-Tasks

### PR DESCRIPTION
**Pull request by JohnTortugo**
Relates to: https://github.com/dotnet/arcade/issues/2913

This is basically a copy of the release pipeline .proj files, that are under \eng\common currently, to the SdkTasks folder. With the move to YAML pipelines those files will be used as sdk-tasks.

### CODE REVIEW

File: src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SigningValidation.proj [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      <!--
+        Documentation for these arguments is available here:
+        https://github.com/dotnet/arcade/tree/master/src/SignCheck
+      -->
+      <SignCheckArgs Include="--recursive" />
+      <SignCheckArgs Include="--traverse-subfolders" />
+      <SignCheckArgs Include="--file-status AllFiles" />
+      <SignCheckArgs Include="--log-file $(SignCheckLog)" />
+      <SignCheckArgs Include="--error-log-file $(SignCheckErrorLog)" />
+      <SignCheckArgs Include="--input-files $(SignCheckInputDir)" />
```
Comment (tmat) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SigningValidation.proj]: Shouldn't the argument values be quoted as well?

File: src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/PublishToPackageFeed.proj [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+    <!--
+      For now the type of packages being published will be informed for the whole build.
+      Eventually this will be specified on a per package basis:
+      TODO: https://github.com/dotnet/arcade/issues/2266
+    -->
+    <PropertyGroup>
+      <TargetStaticFeed Condition="'$(ArtifactsCategory.ToUpper())' == '.NETCORE'">https://dotnetfeed.blob.core.windows.net/cesar-test/index.json</TargetStaticFeed>
+      <TargetStaticFeed Condition="'$(ArtifactsCategory.ToUpper())' == '.NETCOREVALIDATION'">https://dotnetfeed.blob.core.windows.net/cesar-test2/index.json</TargetStaticFeed>
+      <TargetStaticFeed Condition="'$(ArtifactsCategory.ToUpper())' == '.NETCORESTABLE'">https://dotnetfeed.blob.core.windows.net/cesar-test3/index.json</TargetStaticFeed>
```
Comment (tmat) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/PublishToPackageFeed.proj]: `cesar-test`?

Comment (JohnTortugo) [reply src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/PublishToPackageFeed.proj]: Fixed :-/

File: src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SigningValidation.proj (hunk 1 above)
Comment (JohnTortugo) [reply src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SigningValidation.proj]: Well, that wasn't needed so far.

Comment (JohnTortugo) [reply src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SigningValidation.proj]: Fixed

**Pull request merged**